core/src/checks/verify_no_rollback.c

```diff
@@ -62,3 +71,26 @@ int verify_no_rollback(void) {
   INFO("verify_no_rollback: ok");
   return 0;
 }
+
+/**
+ * This hard-coded test case verifies that no_rollback_write_to_buf() produces a known output from the current
+ * VERSION_MAGIC and VERSION constants. It needs to be updated when the VERSION is incremented.
+ */
+int verify_no_rollback_write_to_buf(void) {
```

Comment:
oh I see it in core/src/no_rollback.c

Reply:
Yeah it's because the next commit changes the logic, so I wanted a hard-coded self-check for that function to verify that changing the format identifiers from `"%d"` to `"%d" PRIu32` didn't change the resulting string.